### DISCUSSION TITLE
Rename Provisioning Entries

### DISCUSSION THREAD

**Discussion by MarkusH1975** [zwave-js/zwave-js-ui#3694] [Q&A] [answered]
Hi, 

how can I rename provisioning entries after paring?

Thanks,
Markus

**Comment by robertsLando** [comment]
You can't, you firstly have to remove the node

**Reply by MarkusH1975** [reply, marked as answer]
I found a way to do it.
Export as json file.
Edit json-file.
Import json file.

**Reply by MarkusH1975** [reply]
@robertsLando 
What I want to achieve, is to syncronize the provisioning name with the device name. (and location)
Maybe as an feature idea there could be an option to sync these names from the device to the provisioning entry.

**Reply by robertsLando** [reply]
https://github.com/zwave-js/zwave-js-ui/issues/3699
